tests/cases/fourslash/fourslash.ts

```diff
@@ -240,6 +242,7 @@ declare namespace FourSlashInterface {
         getSyntacticDiagnostics(expected: string): void;
         getSemanticDiagnostics(expected: string): void;
         ProjectInfo(expected: string[]): void;
+        allRangesAppearInImplementationList(): void;

```

Comment:
Since this is always preceded by `goTo.marker()`, why not make that a parameter?